.github/workflows/deploy.yaml

```diff
@@ -135,7 +135,7 @@ jobs:
       - uses: actions/checkout@v2
         with:
           repository: nguyen102/plan_executor
-          ref: r4-aws-fhir-solution
+          ref: r4-aws-fhir-fixed-search
```

Comment:
I think that we should merge the fix into the `r4-aws-fhir-solution` of `nguyen102/plan_executor` and not modify this file

Reply:
Gotcha, I'll make a PR to merge changes into the `r4-aws-fhir-solution` branch instead, and close this PR.